src/Stryker.RegexMutators/Stryker.RegexMutators/Stryker.RegexMutators.csproj

```diff
@@ -2,6 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AssemblyVersion>0.22.9.0</AssemblyVersion>
```

Comment:
In #1681 @richardwerkman proposes to simply remove all these version numbers since they are not used anywhere anyway. They would default and remain static but this should not really be a problem afaik.

Reply:
done
I rebased the branch to master head as it was behind